### PR TITLE
Update to Unicode v9.0.0

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -245,13 +245,13 @@ function prepare() {
...
 		}
 	}
-		
-	unicode.blocks.forEach(function (unicode_block) {
+
+	unicode.Block.forEach(function (unicode_block) {
 		UnicodeBlocks.push({
 			name: unicode_block,
-			regex: require('unicode-7.0.0/blocks/' + unicode_block + '/regex')
+			regex: require('unicode-8.0.0/Block/' + unicode_block + '/regex')
```
Comment (DavidBruant) [inline index.js]: Did you mean `unicode-9.0.0`?

Comment (mathiasbynens) [inline index.js]: Good catch! Fixed :)

**Comment by DavidBruant** [comment]
For reference, I used the repo with this PR in yesterdayand it worked as intended for me.

**Comment by sacharen** [comment]
Same for me. I used the repo and it worked.

**Pull request merged**